tools/dragen-germline/3.9.3/dragen-germline__3.9.3.cwl

```diff
@@ -0,0 +1,714 @@
+      - pattern: ".tbi"
+        required: true
+    inputBinding:
+      prefix: "--vc-forcegt-vcf"
+  # cnv pipeline - with this we must also specify one of --cnv-normal-b-allele-vcf, 
+  # More info at https://support-docs.illumina.com/SW/DRAGEN_v39/Content/SW/DRAGEN/CNVExamples_fDG_dtREF.htm?Highlight=cnv-normal-b-allele-vcf
+  enable_cnv:
+    label: enable cnv calling
+    doc: |
+      Enable CNV processing in the DRAGEN Host Software.
+    type: boolean?
+    inputBinding:
+      prefix: --enable-cnv
+      valueFrom: "$(self.toString())"
+  cnv_enable_self_normalization:
+    label: cnv enable self normalization
+    doc: |
+      Enable CNV self normalization. 
+      Self Normalization requires that the DRAGEN hash table be generated with the enable-cnv=true option.
+    type: boolean?
+    inputBinding:
+      prefix: --cnv-enable-self-normalization
+      valueFrom: "$(self.toString())"
+  # QC options
+  qc_coverage_region_1:
+    label: qc coverage region 1
+    doc: |
+      Generates coverage region report using bed file 1.
+    type: File?
+    inputBinding:
+      prefix: --qc-coverage-region-1
+  qc_coverage_region_2:
+    label: qc coverage region 2
+    doc: |
+      Generates coverage region report using bed file 2.
+    type: File?
+    inputBinding:
+      prefix: --qc-coverage-region-2
+  qc_coverage_region_3:
+    label: qc coverage region 3
+    doc: |
+      Generates coverage region report using bed file 3.
+    type: File?
+    inputBinding:
+      prefix: --qc-coverage-region-3
+  qc_coverage_ignore_overlaps:
+    label: qc coverage ignore overlaps
+    doc: |
+      Set to true to resolve all of the alignments for each fragment and avoid double-counting any 
+      overlapping bases. This might result in marginally longer run times. 
+      This option also requires setting --enable-map-align=true.
+    type: boolean?
+    inputBinding:
+      prefix: --qc-coverage-ignore-overlaps
+      valueFrom: "$(self.toString())"
+  # TMB options
+  enable_tmb:
+    label: enable tmb
+    doc: |
+      Enables TMB. If set, the small variant caller, Illumina Annotation Engine, 
+      and the related callability report are enabled.
+    type: boolean?
+    inputBinding:
+      prefix: --enable-tmb
+      valueFrom: "$(self.toString())"
+  tmb_vaf_threshold:
+    label: tmb vaf threshold
+    doc: |
+      Specify the minimum VAF threshold for a variant. Variants that do not meet the threshold are filtered out. 
+      The default value is 0.05.
+    type: float?
+    inputBinding:
+      prefix: --tmb-db-threshold
+  tmb_db_threshold:
+    label: tmb db threshold
+    doc: |
+      Specify the minimum allele count (total number of observations) for an allele in gnomAD or 1000 Genome 
+      to be considered a germline variant.  Variant calls that have the same positions and allele are ignored
+      from the TMB calculation. The default value is 10.
+    type: int?
+    inputBinding:
+      prefix: --tmb-db-threshold
```

Comment:
Would these be appropriate for the germline caller? I think TMB is done only in tumor-only or tumor-normal mode of the sv caller - https://support-docs.illumina.com/SW/DRAGEN_v39/Content/SW/DRAGEN/Biomarkers_TMB.htm